client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -463,7 +464,9 @@ export const ApplicationsTable: React.FC = () => {
     if (applicationWriteAccess) {
       actions.push({
         title: t("actions.delete"),
-        isDisabled: row.migrationWave !== null,
+        isAriaDisabled: row.migrationWave !== null,
```

Comment:
Are we confident that the wave value is null after deletion? Maybe it is undefined?

Reply:
Maybe a case of stale applications data after wave deletion?

Reply:
Maybe invalidate the applications query on wave delete @gildub or refetch apps on wave delete.

Reply:
Hmmm so we're going to fall back to filter() to see if that more robust in this case.

Reply:
oh filter/some won't make a difference if our query cache is out of date.

Reply:
Yeah I suppose. Just that I didn't experience any issues while testing but that might be just my test set.

Reply:
Did you not see the issue?
> @ibolton336, regarding [#968 (comment)](https://github.com/konveyor/tackle2-ui/pull/968#issuecomment-1573949475), I can see the issue but not sure why this is happening.

Reply:
Yes I can see the issue when tooltip is showing even though dropdown is enabled.
It's happening all the time no matter what the data, so my confusion.

Reply:
![image](https://github.com/konveyor/tackle2-ui/assets/1901741/04e3e1c3-0444-47a2-a653-ff2dd4f35a44)

Reply:
Understood. I thought we were on to the next suspect.